### DISCUSSION TITLE
Encyrpted External Drive

### DISCUSSION THREAD

**Discussion by kadircanersahin** [stashapp/stash#3641] [Q&A] [answered]
Stash can't find encrpted media path from external HDD, getting "file does not exist" or "The system cannot find the path specified." errors.
(User: Noob, OS: Windows, App: Cryptomator)

**Comment by MrX292** [comment]
did you use the drive path you get when you reveal drive after unlocking the vault?

**Comment by kadircanersahin** [comment, marked as answer]
Solved: Do not the Cryptomator. I ran the Cryptomator and Stashapp normally, not "as administrator". Worked for another media servers too. Thanks. [*](https://duckduckgo.com/?q=sorry+for+my+bad+english)
